src/systems/ec/implementations/vanilla/packages/ec-component-form-text-area/ec-component-form-text-area.scss

```diff
@@ -25,27 +23,56 @@
     display: block;
     font: $ecl-font-m;
     margin: 0;
-    max-width: $max-width;
-    min-width: $min-width;
+    max-width: 100%;
     padding: calc(#{$padding-vertical} - #{$border-width})
       calc(#{$padding-horizontal} - #{$border-width});
     width: 100%;
 
+    &:hover {
+      outline: 1px solid $ecl-color-blue-100;
+      outline-offset: -1px;
+    }
+
+    &:active,
     &:focus {
       outline: $border-width-focus solid $border-color-focus;
       outline-offset: -3px;
     }
   }
 
+  .ecl-text-area--md {
```

Comment:
we should use the same naming convention for classes as in other components.
so `ecl-text-area--s`, `ecl-text-area--m`, `ecl-text-area--l`